exo-go/bin/lint

```diff
@@ -8,5 +8,5 @@ fi
 
 # required by gotype to work properly
 # see https://github.com/golang/go/issues/12703#issuecomment-144429128
-go install
-gometalinter.v1 --deadline 60s ./
+go install ./...
+gometalinter.v1 --deadline 60s ./...
```

Comment:
The vendor folder is on the top level instead of in exo-go, thus we are already avoiding that